README.md

```diff
@@ -3,90 +3,121 @@ GOV.UK Frontend ·
+Below is a code sample you can add to your gulp configuration file using the gulp-sass package:
+`.pipe(sass({ includePaths: 'node_modules/' }))`
 
-Install a button component
+To import images, configure your application to reference or copy the icons assets.
 
-```
-npm install --save @govuk-frontend/button
-```
+Below is a code sample you can add to your [express.js](https://expressjs.com/) configuration:
+`app.use('/public', express.static(path.join(__dirname, '/node_modules/@govuk-frontend/icons')))`
 
-All components can be found in the packages directory.
-Each component has a README with installation and usage instructions.
+## Usage
+
+Copy and paste code from the examples in the GOV.UK [Design System](https://govuk-design-system-production.cloudapps.digital/)  to use GOV.UK Frontend in your service.
+
+<a id="option2"></a> 
+
+# Option 2: Add assets manually
 
-## Quick start (using all components)
+## Download assets
 
-:rotating_light: **This will not work until packages are published to npm.** :rotating_light:
+Download the latest versions of the following assets and include them in your project:
 
-Include the CSS and JavaScript
+- [govuk-frontend/tree/master/dist/css](https://github.com/alphagov/govuk-frontend/tree/master/dist/css)
+- [govuk-frontend/tree/master/dist/js](https://github.com/alphagov/govuk-frontend/tree/master/dist/js)
+- [icons](https://github.com/alphagov/govuk-frontend/tree/master/dist/icons)
 
-```
+## Include assets
+
+Add the CSS and JavaScript code to your HTML template:
+
+~~~
 <!DOCTYPE html>
 <html>
   <head>
     <title>Example</title>
-    <link rel="stylesheet"
-          href="node_modules/@govuk-frontend/all/all.min.css">
+    <link rel="stylesheet" href="assets/govuk-frontend-[latest version]-alpha.min.css">
+    <!--[if IE 8]>
+      <link rel="stylesheet" href="assets/govuk-frontend-oldie-[latest version]-alpha.min.css">
+    <![endif]-->
   </head>
   <body>
     <!-- Copy and paste component HTML-->
     <button class="govuk-c-button">This is a button component</button>
-    <script src="node_modules/@govuk-frontend/all/all.min.js"></script>
+    <script src="assets/govuk-frontend-[latest version]-alpha.min.js"></script>
   </body>
 </html>
-```
-
-## Usage
+~~~
+## Usage 
 
-## HTML
+Copy and paste code from the examples in the GOV.UK [Design System](https://govuk-design-system-production.cloudapps.digital/)  to use GOV.UK Frontend in your service.
 
-Copy and paste component HTML [from here](TODO: Insert link to GOV.UK Design System).
-
-## SCSS
-
-In your main.scss file
-
-```
-// All components
-@import "@govuk-frontend/all/all";
-
-// Pick and choose components
-@import "@govuk-frontend/button/button";
-@import "@govuk-frontend/input/input"
-```
-
-## Docs
-
-Check out the [GOV.UK Frontend documentation](/docs/index.md).
 
 ## Licence
 
-[MIT license](LICENSE.txt).
+This project is licensed under the [MIT License](https://github.com/angular/angular.js/blob/master/LICENSE).
```

Comment:
Should be linking to our own LICENSE.txt.

Reply:
Done